clients/fides-js/src/lib/cookie.ts

```diff
@@ -217,3 +217,31 @@ export const saveFidesCookie = (cookie: FidesCookie) => {
     CODEC
   );
 };
+
+export const setConsentCookieAcceptAll = (defaults: CookieKeyConsent): void => {
```

Comment:
nit: I still don't know what a good name for this function is and I think it's a sign 😓 

Maybe `acceptAllAndSaveFidesCookie`?